### ISSUE TITLE
TypeError: undefined is not a function

### ISSUE THREAD

**Issue by dy** [code42day/postcss-cli#13]
The latest version fails on cssnext:

```
$ postcss -u cssnext empty.css
[TypeError: undefined is not a function]
```

Unfortunately, no debugging info provided.

@MoOx pls, take a look, this seems to happen with the new version of cssnext.

There probably has to be some regression test.

**Comment by MoOx** [comment]
Can you give me which version is working and which one isn't please ?

**Comment by MoOx** [comment]
Can you try with cssnext 1.5.2 please ?

**Comment by MoOx** [comment]
Hum, I think I am getting it without postcss-cli.
Investigating.

**Comment by MoOx** [comment]
Confirming it's another regression related to autoprefixer 5.2 (poke @ai)

**Comment by MoOx** [comment]
Hum well sorry for the noise. PEBCAK. It was the same regression that has already been fixed in 1.5.2. You can close this issue.

**Comment by pirxpilot** [comment]
Closing per @MoOx comment.
